samples/v1/addSecurityMarks.js

```diff
@@ -18,7 +18,7 @@
  * Demostrates adding security marks to an asset.
  */
 function main(assetName = 'full asset path to add marks to') {
-  // [START demo]
+  // [START demo]//  [START securitycenter_demo]
```

Comment:
this looks a little off why is this commented out?

Reply:
Fixed in latest commits!